Makefile

```diff
@@ -30,98 +30,130 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
-all: manager
+.DEFAULT_GOAL := help
 
+help:
+	@awk '/^#/{c=substr($$0,3);next}c&&/^[[:alpha:]][[:alnum:]_-]+:/{print substr($$1,1,index($$1,":")),c}1{c=0}' $(MAKEFILE_LIST) | column -s: -t
```

Comment:
The help that was in place before seemed simpler:
```
help : Makefile
	@sed -n 's/^##//p' $<
```

:)

Reply:
Also, if the comments are multi-lined, it only picks the last line

Reply:
It's a trade off. The previous implementation also required that the indentation of the help messages be handled in the comments themselves, which made it painful to align the descriptions throughout the file. Also it was more verbose as it required the target to be specified in the comment.

IMO detailed usage instructions and explanations belong in the README.md and the `help` text should just be a brief summary of what the command does.

Reply:
ok @ryanemerson fair enough